### PR TITLE
Add Corporate Summary for Organization Owners

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: portal-frontend/src/app/features/edit-application/application-owner-dialog/application-owner-dialog.component.ts [hunk 1]
```diff
@@ -81,10 +94,20 @@ export class ApplicationOwnerDialogComponent {
       console.error('ApplicationOwnerDialogComponent misconfigured, needs fileId for create');
       return;
     }
+
+    let documentUuid;
```
Comment (mhuseinov) [inline portal-frontend/src/app/features/edit-application/application-owner-dialog/application-owner-dialog.component.ts:98]: maybe a private function

File: portal-frontend/src/app/features/review-application/review-attachments/review-attachments.component.ts [hunk 2]
```diff
@@ -20,7 +16,7 @@ export class ReviewAttachmentsComponent implements OnInit, OnDestroy {
 
   $destroy = new Subject<void>();
 
-  documentTypes = APPLICATION_DOCUMENT;
+  documentTypes = DOCUMENT;
```
Comment (mhuseinov) [inline portal-frontend/src/app/features/review-application/review-attachments/review-attachments.component.ts:19]: 👍

File: portal-frontend/src/app/services/application-owner/application-owner.service.ts [hunk 3]
```diff
@@ -100,4 +106,48 @@ export class ApplicationOwnerService {
     }
     return 0;
   }
+
+  async uploadCorporateSummary(files: File[]) {
+    if (files.length > 0) {
+      const file: File = files[0];
+
+      if (file.size > environment.maxFileSize) {
```
Comment (mhuseinov) [inline portal-frontend/src/app/services/application-owner/application-owner.service.ts:114]: definitely outside of this MR. But I feel like we need some more generic document service that could hold this logic and accept DTO. I think we have similar code now in 3 locations

File: services/apps/portal/src/common/automapper/application-owner.automapper.profile.ts [hunk 4]
```diff
@@ -26,6 +28,22 @@ export class ApplicationOwnerProfile extends AutomapperProfile {
...
+        forMember(
+          (ad) => ad.corporateSummary,
+          mapFrom((a): ApplicationDocumentDto | undefined => {
+            if (a.corporateSummary) {
+              return {
+                uuid: a.corporateSummary.uuid,
+                fileName: a.corporateSummary.fileName,
+                type: DOCUMENT_TYPE.CORPORATE_SUMMARY,
+                fileSize: a.corporateSummary.fileSize,
+                uploadedAt: 1,
```
Comment (mhuseinov) [inline services/apps/portal/src/common/automapper/application-owner.automapper.profile.ts:40]: is this '1' intentional?

**Review by mhuseinov — APPROVED** [review]

File: portal-frontend/src/app/services/application-owner/application-owner.service.ts (hunk 3 above)
Comment (dhaselhan) [reply portal-frontend/src/app/services/application-owner/application-owner.service.ts]: Done!  Cleaned up to emit a single file for each one selected, and added a new upload to DRY this up

File: services/apps/portal/src/common/automapper/application-owner.automapper.profile.ts (hunk 4 above)
Comment (dhaselhan) [reply services/apps/portal/src/common/automapper/application-owner.automapper.profile.ts]: No, updated to use createdAt

File: portal-frontend/src/app/features/edit-application/application-owner-dialog/application-owner-dialog.component.ts (hunk 1 above)
Comment (dhaselhan) [reply portal-frontend/src/app/features/edit-application/application-owner-dialog/application-owner-dialog.component.ts]: Done!

**Review by mhuseinov — APPROVED** [review]
🔥

**Pull request merged**